lib/dry/web/roda/generators/abstract_project.rb

```diff
@@ -0,0 +1,52 @@
+require "dry/web/roda/generate"
+
+module Dry
+  module Web
+    module Roda
+      module Generators
+        class AbstractProject
+          attr_reader :target_dir, :options
+
+          def initialize(target_dir, options = {})
+            @target_dir = target_dir
+            @options = options
+            @generator = Generate.new(destination)
+            @templates = []
```

Comment:
I like the idea of avoiding mutating an ivar, I will look for options to do that 👍